src/strtree.c

```diff
@@ -145,7 +147,11 @@ static PyObject* STRtree_new(PyTypeObject* type, PyObject* args, PyObject* kwds)
   n = PyArray_SIZE((PyArrayObject*)arr);
 
   kv_init(_geoms);
+  kv_init(_tree_indexes);
```

Comment:
```suggestion
  _tree_indexes = (npy_intp*)malloc(n * sizeof(npy_intp));
```
(and drop following line)